icu4c/source/common/ucln_cmn.cpp

```diff
@@ -31,6 +31,7 @@
 static cleanupFunc *gCommonCleanupFunctions[UCLN_COMMON_COUNT];
 static cleanupFunc *gLibCleanupFunctions[UCLN_COMMON];
 
+using namespace icu;
```

Comment:
Qualify instead of using namespace?

Wrap the whole file in U_NAMESPACE_BEGIN?

If you have using, you should use the macro U_USING_NAMESPACE

Reply:
Changed to qualifying the uses.